liblangutil/ErrorReporter.cpp

```diff
@@ -28,6 +28,20 @@ using namespace std;
 using namespace solidity;
 using namespace solidity::langutil;
 
+namespace solidity::langutil {
+
+std::string quote(std::string const& _name)
+{
+	return "\"" + _name + "\"";
+}
+
+std::string quoteSpace(std::string const& _name)
+{
+	return " " +quote(_name) + " ";
```

Comment:
```suggestion
	return " " + quote(_name) + " ";
```
Please also make an addition to the style checker.